### ISSUE TITLE
Improve decision making process with simpler form and enablement of PR

### ISSUE THREAD

**Issue by derberg** [kyma-project/community#241]
Created on 2019-03-20 by @derberg.

## Decision log

| Name | Description |
|-----------------------|------------------------------------------------------------------------------------|
| Title | Make decision log shorter and allow to use it on PRs |
| Due date | 2019-03-27 |
| Status | Proposed |
| Affected decisions | Initial setup of decision making process. |

## Context

We got feedback that when creating proposals and creating decisions log, the format of the log is overwhelming. It is also sometimes not intuitive to make a separate issue with decision log even though you know your proposal is simple, there will be no contra-proposal and decision log could be done in PR description

### Decision log template
Remove following sections from current decision log:
- Ultimate decision maker -> it is always Kyma council, we just need to write in in governance doc
- Input provider(s) -> anyone can be an input provider
- Group(s) affected by the decision -> we do not really have any open definition of groups so it is hard to specify, even for this decision record I was not sure what to put
- Earliest date to revisit the decision -> something that we might need in future maybe but now it is also confusing and always hard to specify, there are no clear conditions to specify such date

### Decision log in PR

We want to also make it possible for a proposal maker to include the decision log inside his PR so he doesn't have to go to the issue separately once his PR is merged.

## Consequences

- Happy people
- Cleanup of docs needed
  - update https://github.com/kyma-project/community/blob/master/.github/ISSUE_TEMPLATE/decision-record.md
  - create new template for PR with decision log
  - delete https://github.com/kyma-project/community/blob/master/guidelines/templates/resources/DR.md
  - update governance model https://github.com/kyma-project/community/blob/master/governance.md

**Comment by agolawska** [comment]
I am ok with the proposal

**Comment by agolawska** [comment]
https://github.com/kyma-project/community/pull/282  ready for review
